src/utils/showModal.js

```diff
@@ -1,8 +1,8 @@
-function showModal(header, callback) {
+function showModal(title, header, callback) {
```

Comment:
A bit confusing names **title** and **header**. Not clear what is what. May be **title** and **contentText**? It will match Dialog components names.
But not a blocker for this PR. We can update names later.